test/e2e-tests/README.md

```diff
@@ -21,14 +21,14 @@ The structure of the `config.json` is in four sections, the top level fields, `g
 - `image_registry_username` - quay.io username
 - `image_registry_password` - quay.io password
 - `target_image_tags` - JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
+- `git_container_file_repo` - Git repo containing the container file
+- `git_container_file_revision` - Git branch in the container file repo
+- `container_relative_path` - Relative path from the root of the cotainer file repo to where the container file is
```

Comment:
```suggestion
- `container_relative_path` - Relative path from the root of the container file repo to where the container file is
```

Reply:
I misspell at least once a pr... thank you

Reply:
@jackdelahunt no worries, with this amount of text it happens.